bigquery/README.md

```diff
@@ -4,3 +4,54 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+## Accessing build insights with SQL queries through the BigQuery CLI:
+
+```bash
+$ bq query '<SQL QUERY>'
+```
+Legacy SQL dialect is set on default for the BigQuery CLI and must be disabled for the example queries to work.
+This can be done by editing ~/.bigqueryrc and adding the following lines:
+
+```
+[query]
+--use_legacy_sql=false
+```
+More information can be found [here](https://cloud.google.com/bigquery/docs/reference/standard-sql/enabling-standard-sql).
+
+Example Queries:
+```sql
+# Listing overall build history
+
+SELECT * FROM `dataset.table`
```

Comment:
Will the user's table name be `dataset.table`? Or do we need to tell them how to discover their table name?

Reply:
That would be nice, but even better would be linking to any bq CLI docs 😄

Reply:
https://cloud.google.com/bigquery/docs/bq-command-line-tool << maybe this one?